specs/referrer-policy/index.src.html

```diff
@@ -516,18 +516,18 @@ <h3 id="referrer-policy-delivery-meta">Delivery via <{meta}></h3>
   </dl>
 
   <h3 id="referrer-policy-delivery-referrer-attribute">Delivery
-  via a <code>referrer</code> attribute</h3>
+  via a <code>referrerpolicy</code> attribute</h3>
 
-  A referrer policy may be set when a <code>referrer</code> attribute is
+  A referrer policy may be set when a <code>referrerpolicy</code> attribute is
```

Comment:
the spec should also include what happens. e.g. if the user clicks on the link, but it has an onclick handler that then creates an XHR, does the referrer policy apply to the XHR as well?

What if it uses window.open() to navigate, or assign a value to location.href?